integrations.go

```diff
@@ -158,6 +158,51 @@ func (client *Client) DeleteIntegrationSlack() error {
 	return client.doJsonRequest("DELETE", "/v1/integration/slack", nil, nil)
 }
 
+/*
+	Webhook Integration
+*/
+
+// IntegrationWebhookRequest defines the structure of the a webhook request
+type IntegrationWebhookRequest struct {
+	Webhooks []Webhook `json:"hooks"`
+}
+
+// Webhook defines the structure of the a webhook
+type Webhook struct {
+	Name             *string `json:"name"`
+	URL              *string `json:"url"`
+	UseCustomPayload *string `json:"use_custom_payload"`
+	CustomPayload    *string `json:"custom_payload"`
+	EncodeAsForm     *string `json:"encode_as_form"`
+	Headers          *string `json:"headers"`
+}
+
+// CreateIntegrationWebhook creates a new Webhook Integrations.
```

Comment:
I'd suggest using:

```
// CreateIntegrationWebhook creates new webhook integration object(s).
```

Reply:
I have changed this, but this method does not create/return an object but actually creates on in Datadog..